add2cal/add2cal.py

```diff
@@ -11,12 +11,12 @@
 
 BASE_URLS = {
     'google': 'https://calendar.google.com/calendar/render',
-    'outlook': 'https://outlook.office.com/owa/',
+    'outlook': 'https://outlook.office.com/calendar/action/compose',
     'yahoo': 'http://calendar.yahoo.com'
 }
 
 INPUT_DATE_FORMAT = "%Y%m%dT%H%M%S"
-OUTLOOK_DATE_FORMAT = '%Y-%m-%dT%I:%M:%SZ'
+OUTLOOK_DATE_FORMAT = '%Y-%m-%dT%I:%M:%S'
```

Comment:
why remove the timezone info?

Reply:
we have the dates already in the user's preferred timezone is available or UTC. If added, the outlook timezone displays a different unexpected time